test/unit/test_command_wrapper.py

```diff
@@ -3,24 +3,23 @@
 from test import HokusaiUnitTestCase
 from test.utils import captured_output
 
-from hokusai.lib.command import command
+from hokusai.lib.command_wrapper import wrap
```

Comment:
I'm not sure why we need to change the package and the semantics of this function... `wrap` seems too generic to me. Is there something wrong with just keeping it as `command`? Either that or `execute` would seem to me more meaningful than `wrap`.

Reply:
yeah let's stick with `command`. updated.